### PR TITLE
Fixes Typos

### PR DESCRIPTION
**Pull request by christophherr**
Hi,

I just translated the plugin at translate.wordpress.org and found a few typos.

### CODE REVIEW

**Comment by pdewouters** [comment]
Thanks @christophherr , changes look good!

File: admin/faq.php [hunk 1]
```diff
@@ -29,11 +29,11 @@
 
      '<p><strong>' . __( 'How long should a backup take?', 'backupwordpress' ) . '</strong></p>' .
 
-     '<p>' . __( 'Unless your site is very large (many gigabytes) it should only take a few minutes to perform a back up, if your back up has been running for longer than an hour it\'s safe to assume that something has gone wrong, try de-activating and re-activating the plugin, if it keeps happening, contact support.', 'backupwordpress' ) . '</p>' .
+     '<p>' . __( 'Unless your site is very large (many gigabytes) it should only take a few minutes to perform a back up. If your back up has been running for longer than an hour, it\'s safe to assume that something has gone wrong. Try de-activating and re-activating the plugin. If it keeps happening, contact support.', 'backupwordpress' ) . '</p>' .
```
Comment (pdewouters) [inline admin/faq.php]: should actually be `backup` in one word, if you'd like to fix that in the same PR

File: admin/faq.php [hunk 2]
```diff
@@ -29,11 +29,11 @@
...
 
      '<p><strong>' . __( 'What do I do if I get the wp-cron error message?', 'backupwordpress' ) . '</strong></p>' .
 
-     '<p>' . __( 'The issue is that your <code>wp-cron.php</code> is not returning a <code>200</code> response when hit with a HTTP request originating from your own server, it could be several things, in most cases, it\'s an issue with the server / site.', 'backupwordpress' ) . '</p>' .
+     '<p>' . __( 'The issue is that your <code>wp-cron.php</code> is not returning a <code>200</code> response when hit with a HTTP request originating from your own server. It could be several things. In most cases, it\'s an issue with the server / site.', 'backupwordpress' ) . '</p>' .
```
Comment (pdewouters) [inline admin/faq.php]: I think `, it could be several things` with a comma is ok here

File: admin/faq.php (hunk 1 above)
Comment (christophherr) [inline admin/faq.php]: Thank you! I was wondering about it. I tend to stay away from any words that are contained in the name of the plugin. I´ll fix it in the PR.

File: admin/faq.php (hunk 2 above)
Comment (christophherr) [inline admin/faq.php]: Yes, it is ok. I just tried to shorten the sentence. I´ll update the PR.

File: admin/faq.php [hunk 3]
```diff
@@ -55,4 +55,4 @@
 
      '<p><strong>' . __( 'My backups seem to be failing?', 'backupwordpress' ) . '</strong></p>' .
 
-     '<p>' . __( 'If your backups are failing, it\'s commonly caused by a lack of available resources on your server. The easiest way to establish this is to exclude some of, or the entirety of your uploads folder, running a backup, and if that succeeds, then you\'ll know it\'s probably a server issue. If not, report the results to our support team for further help. To do this, either enable support from your Admin Dashboard (recommended), or email backupwordpress@hmn.md', 'backupwordpress' ) . '</p>';
+     '<p>' . __( 'If your backups are failing, it\'s commonly caused by a lack of available resources on your server. To establish this is the case, exclude the complete (or parts of the) uploads folder and run a backup. If that succeeds, you know it\'s probably a server issue. If it does not succeed, report the results to our support team for further help. You can contact support by enabling support from your Admin Dashboard (recommended), or emailing backupwordpress@hmn.md', 'backupwordpress' ) . '</p>';
```
Comment (pdewouters) [inline admin/faq.php]: I'm not sure this is an improvement? cc @willmot

Comment (willmot) [inline admin/faq.php]: I think it's better.

Comment (pdewouters) [inline admin/faq.php]: ok!

**Comment by pdewouters** [comment]
thanks @christophherr

**Comment by christophherr** [comment]
Thanks for a great plugin!

**Pull request merged**